plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.tsx

```diff
@@ -75,20 +82,26 @@ export const RecentWorkflowRunsCard = ({
   const githubHost = hostname || 'github.com';
 
   return !runs.length ? (
-    <EmptyState
-      missing="data"
-      title="No Workflow Data"
-      description="This component has GitHub Actions enabled, but no data was found. Have you created any Workflows? Click the button below to create a new Workflow."
-      action={
-        <Button
-          variant="contained"
-          color="primary"
-          href={`https://${githubHost}/${owner}/${repo}/actions/new`}
-        >
-          Create new Workflow
-        </Button>
-      }
-    />
+    <Card>
```

Comment:
probably want InfoCard here too just to make sure that things turn out consistent? and skip the divider probably then

Reply:
Hey I have a suggestion. Let's skip this large empty state entirely.

Down on the line that renders the table, we could instead do

```ts
{!runs.length ? (
  <>
    This component has GitHub Actions enabled, but no workflows were found.
    <Link to="...">Create a new workflow</Link>
  </>
) : (
  <Table ...
```

i am starting to feel that these large empty states are a poor fit for the cards and pages where there are several different things going on

Reply:
This looks awesome!

This may be tangential and possibly useful for a separate PR, but one thing we need to be careful about is currently (I believe) the "GItHub Actions" enablement is based solely on the existence of the annotation `github.com/project-slug`. This isn't technically fully true, in that you might have a repo with no actions enabled.  In future, this plugin could be useful to add a check to the API to validate if the repo is even ENABLED for actions:
https://docs.github.com/en/rest/reference/actions#get-github-actions-permissions-for-a-repository

If enabled, then the card visual above about "create a new workflow" is perfect. Otherwise, we might instead opt to NOT show this card at all and the annotation therefore "disabled" and no GitHub-actions plugin for this entity. Makes it a bit more dynamic.  (and e.g. for my space, we'll use CI in a different tool than actions, and so don't want missing actions showing up every time we view an entity)

Otherwise visually, I'm fully in alignment with the discussion through here!